solidity/contracts/bridge/BridgeState.sol

```diff
@@ -61,20 +61,8 @@ library BridgeState {
         //
         // This is a per-deposit input max fee for the sweep transaction.
         uint64 depositTxMaxFee;
-        // Collection of all revealed deposits indexed by
-        // `keccak256(fundingTxHash | fundingOutputIndex)`.
-        // The `fundingTxHash` is `bytes32` (ordered as in Bitcoin internally)
-        // and `fundingOutputIndex` an `uint32`. This mapping may contain valid
-        // and invalid deposits and the wallet is responsible for validating
-        // them before attempting to execute a sweep.
-        mapping(uint256 => Deposit.DepositRequest) deposits;
-        // Indicates if the vault with the given address is trusted or not.
-        // Depositors can route their revealed deposits only to trusted vaults
-        // and have trusted vaults notified about new deposits as soon as these
-        // deposits get swept. Vaults not trusted by the Bridge can still be
-        // used by Bank balance owners on their own responsibility - anyone can
-        // approve their Bank balance to any address.
-        mapping(address => bool) isVaultTrusted;
+        // slither-disable-next-line unused-state
+        bytes8 __depositAlignmentGap;
```

Comment:
Could we add a comment why we added this variable?
(I'm guessing it's to make the layout of variables more efficient).